extensions/wikia/PortabilityDashboard/templates/SpecialPortabilityDashboard_index.mustache

```diff
@@ -1,3 +1,15 @@
+<section class="Search clearfix">
+    <form class="WikiaSearch" id="search-v2-form" action="">
+        <div class="SearchInput">
+            <p class="grid-1 alpha">{{searchHeadline}}</p>
+            <input type="text" name="url" id="search-v2-input" class="search-v2-input" placeholder="{{searchPlaceholder}}">
+            <button type="submit" class="wikia-button" id="search-v2-button" value="Search">
+                <img src="{{blankImgUrl}}" class="sprite search" height="17" width="21">
+            </button>
```

Comment:
do we need all this classes and id's: `search-v2-form`, `search-v2-input`, `search-v2-button` `sprite search` `alpha`?
Could `sprite search` be styled in `PortabilityDashboard.scss` ?